pkg/stores/proxy/proxy_store.go

```diff
@@ -447,7 +447,8 @@ func (s *Store) Create(apiOp *types.APIRequest, schema *types.APISchema, params
 	input["apiVersion"], input["kind"] = gvk.ToAPIVersionAndKind()
 
 	buffer := WarningBuffer{}
-	k8sClient, err := metricsStore.Wrap(s.clientGetter.TableClient(apiOp, schema, namespace, &buffer))
+	// k8sClient, err := metricsStore.Wrap(s.clientGetter.TableClient(apiOp, schema, namespace, &buffer))
```

Comment:
Can be removed since git will keep history in case we need it back

Reply:
Oops didn't realize I hadn't remove those :expressionless: